node_modules/.bin/yarn

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+
+case `uname` in
+    *CYGWIN*|*MINGW*|*MSYS*) basedir=`cygpath -w "$basedir"`;;
+esac
+
+if [ -x "$basedir/node" ]; then
+  exec "$basedir/node"  "$basedir/../yarn/bin/yarn.js" "$@"
+else 
+  exec node  "$basedir/../yarn/bin/yarn.js" "$@"
+fi
```

Comment:
you should not push node_modules folder in github

Reply:
Oh is it so? I didn't knew that. Thanks for feedback!